Automattic/sensei
# Fix JavaScript error on Lessons page

This PR fixes the following JavaScript error that shows up when browsing to _Lessons_ in WP Admin:

![bulk-lesson-error](https://user-images.githubusercontent.com/1190420/42101173-30f9e642-7bba-11e8-86d1-b917593abf75.jpg)

## Testing
1. Open Chrome DevTools.
2. Browse to _Lessons_ in WP Admin.
3. Ensure the above error does not appear in the Console.
